www/netkisdk.js

```diff
@@ -40,6 +39,10 @@ NetkiSDK.prototype.isAvailable = function (successCallback, errorCallback) {
     exec(successCallback, errorCallback, "NetkiSDK", "isAvailable", []);
 }
 
+NetkiSDK.prototype.configure = function(appToken, serviceCode, successCallback, errorCallback) {
```

Comment:
You do not this method, please reuse the one in line 46, that is the one that we need to expose